common/f3d_materials.md

```diff
@@ -0,0 +1,193 @@
+
+Shade alpha has its own quirks. Shade alpha will be vertex colors unless the `fog` geo mode is enabled, then shade alpha will become the fog value. Fog will change from 0 to 1 depending on how far away the mesh triangles are from the camera. The range of travel depends on the fog intensity chosen.
+
+[LOD Fraction]{.key_text} is a variable that determines how far between LOD levels the current texel will be. This value shifts between 0 and 1 as the camera moves farther away from the target texel. Once reaching one, the value will wrap back to zero until the max LOD level is reached. This value is only enabled while using LOD calculations.
+
+## Color Registers
+
+Color registers are RGBA values that are stored in Display Memory (DMEM) and used for combiner operations. These registers hold a discrete value until they are changed by a new material. The RGB values selected will be used under the alias `<X> Color` while the alpha component will be under `<X> Color Alpha`, where <X> representing the color registers name. All colors are constant rgba colors, though some registers have alternative usages or ad hoc purposes beyond being a free use color.
+
+* Environment Color: Free use register
+
+* Primitive Color: Free use register. Encoded at the same time as LOD variables but those have no impact on the color.
+
+* Chroma Key Center: Made to support chroma key usage. While not using chroma key, simply exists as a free use RGB register.
+
+## Discrete Values
+
+Discrete values are adjustable float values between 0 and 1 that you can use for blending or interpolating colors. These are simply factors that you can use to fine tune the contribution of each input to the color combiner. Some registers are created with specific graphical purpose, while others are just for user flexibility.
+
+* Primitive LOD Fraction This register exists simply for free use
+
+* Chroma Key Scale: Made to support chroma key usage. While not using chroma key, simply exists as a free use RGB register.
+
+* YUV Convert K4, YUV Convert K5: Both registers are made to support YUV to RGB conversion. If not converting, these registers are free to use.
+
+::::
+::::{tab-item} Geo
+
+The geo tab refers to geometry mode selection, which affects how triangles are processed while loading or drawing. Some options require other options to be enabled, or change what values certain variables will refer to. These nuances should be described with tooltips when hovering over the options in fast64.
+
+## Z Buffer
+
+Enabling [Z buffer]{.key_text} will make it so that Z buffer calculations are carried out on drawn triangles. This makes it so that triangles can utilize depth checking per pixel if enabled inside the RDP. Disabling Z buffer will change the drawing algorithm of the triangles sent to the RDP to ignore depth. Depth values are never written to triangles in DMEM and when rasterizing depth is never considered for the entire primitive.
```

Comment:
Nit: Disabling geometry mode Z buffer means Z values are not sent to the RDP, so the Z values used in the tri are garbage. This does not mean depth is not considered, if you have Z enabled in the blender, it'll use the garbage values. I'd replace the last part with saying "if you disable this, make sure Z is not used in the blender in your rendermodes".

Reply:
Is this also true for `z_src_prim`